code/game/gamemodes/heist/heist.dm

```diff
@@ -35,13 +34,12 @@ var/global/list/obj/cortical_stacks = list() //Stacks for 'leave nobody behind'
 	var/list/candidates = get_players_for_role(ROLE_RAIDER)
 	var/raider_num = 0
 
-	//Check that we have enough vox.
+	//Check that we have enough raiders.
 	if(candidates.len < required_enemies)
 		return 0
-	else if(candidates.len < recommended_enemies)
-		raider_num = candidates.len
-	else
-		raider_num = recommended_enemies
+
+
+	raider_num = rand(required_enemies, min(recommended_enemies, candidates.len))//pick a random amount, from the minimum up to the maximum, if numbers permit
```

Comment:
Generally not a good way to balance a game mode; this should either be static and they're given extra gear to compensate for there being more players, orrrr their own number scale with the amount of players.